CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.2.2 (Next)
 
 * [#315](https://github.com/alexa-js/alexa-app/pull/315): Fix card tests and required card type - [@kobim](https://github.com/kobim).
+* [#314](https://github.com/alexa-js/alexa-app/pull/314): Fix breaking logic in request.slot() - [@User1m](https://github.com/user1m).
```

Comment:
ping @dblock

Reply:
Much better! ;) I would prefer that it actually explains the problem. From the code it looks like a slot value is set to `undefined` instead of not being present? Maybe "Fix handling of undefined slot values"? No strong feelings though.